src/Serilog.Enrichers.Sensitive/PathWrapper.cs

```diff
@@ -0,0 +1,19 @@
+using System.IO;
+
+namespace Serilog.Enrichers.Sensitive;
+
+public class PathWrapper : IPathWrapper
```

Comment:
I would strongly recommend to remove this abstraction, it's not adding any value right now as there is no other implementation of `IPathWrapper` and the methods you are using do not actually depend on the filesystem being available.

So I'd prefer that where you're calling `_pathWrapper.GetDirectoryName()`, just call `Path.GetDIrectoryName` instead.

Reply:
The only reason I made such abstraction, cuz I thought if there's a specific when some of the OS unable to get the path properly, then developer might make it own implementation. Let me know if this is still should be removed

Reply:
On the other hand, Path.GetDIrectoryName works weird, sometimes It's unable to get the directory name correct

Reply:
I understand where you're coming from, however you may expect from the runtime to handle this for you.

Reply:
Now path wrapper removed